devnet/node.yaml

```diff
@@ -105,16 +105,20 @@ spec:
             # - http://terra2-terrad:1317
             # - --terra2Contract
             # - terra14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9ssrc8au
-            # - --algorandAppID
-            # - "4"
-            # - --algorandIndexerRPC
-            # - http://algorand:8980
-            # - --algorandIndexerToken
-            # - "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
-            # - --algorandAlgodRPC
-            # - http://algorand:4001
-            # - --algorandAlgodToken
-            # - "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+            - --algorandAppID
+            - "4"
+            - --algorandIndexerRPC
+            - http://algorand:8980
+            - --algorandIndexerToken
+            - "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+            - --algorandAlgodRPC
+            - http://algorand:4001
+            - --algorandAlgodToken
+            - "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+            - --nearRPC
```

Comment:
the algorand stuff should stay commented out and the near stuff can be added in comments, but should be put in the Tiltfile with the other optional chain arguments.

Reply:
What if I want the algorand stuff enabled?

Reply:
It's enabled by default and added to the command parameters over here https://github.com/certusone/wormhole/blob/dev.v2/Tiltfile#L211-L223 which is where you should also add the near stuff 🙂

Reply:
Switch to the new style